### PR TITLE
Redesign Cleanup

### PR DESCRIPTION
**Pull request by falklinus**
## What?
- Fix some first notes/comments on the components

## Why?
- Align/improve codestyle

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: shared/@hedvig-ui/redesign/FillBar/FillBar.tsx [hunk 1]
```diff
@@ -23,13 +23,14 @@ export const FillBar = (props: FillBarProps) => {
   const sum = parts.reduce((acc, { value }) => acc + value, 0)
   const filled = sum / max
   const color = getColorFromAmountFilled(filled)
+  const maxValue = filled > 1 ? sum : max
```
Comment (alebedev) [inline shared/@hedvig-ui/redesign/FillBar/FillBar.tsx:26]: Math.max(sum, max) ?

File: shared/@hedvig-ui/redesign/Grid/Grid.tsx [hunk 2]
```diff
@@ -24,16 +24,12 @@ export const Grid = styled(Div)<GridProps>(
...
-      ? templateColumns
-      : equalColumns
-        ? `repeat(${equalColumns}, minmax(0, 1fr))`
-        : 'initial'};
-    grid-template-rows: ${templateRows
-      ? templateRows
-      : equalRows
-        ? `repeat(${equalRows}, minmax(0, 1fr))`
-        : 'initial'};
+    grid-template-columns: ${templateColumns ?? equalColumns
```
Comment (alebedev) [inline shared/@hedvig-ui/redesign/Grid/Grid.tsx:27]: What is operator precedence here, maybe we can make it explicit?

Something like
```
(templateColumns ?? equalColumns) ? ... : ...
```

**Review by alebedev — APPROVED** [review]
Looks great!

File: shared/@hedvig-ui/redesign/Grid/Grid.tsx (hunk 2 above)
Comment (falklinus) [reply shared/@hedvig-ui/redesign/Grid/Grid.tsx]: I'm making it only available to pass one or the other now

**Pull request merged**